themes/default/content/blog/architecture-as-code-serverless/index.md

```diff
@@ -198,4 +198,4 @@ if (provisionedConcurrentExecutions) {
 
 ## Conclusion
 
-Serverless is an attractive alternative to architectures that require hands-on management of servers. Despite this, it does not relieve you of the tasks needed to create and manage the ancillary resources needed for a modern application in production. Architecture as code enables you to declare resources and reuse them to ensure a standard and reproducible infrastructure. Also, you can deploy multiple functions and applications with a single Pulumi project, saving time and effort required to deploy them individually. Finally, you can tune your deployment strategies to overcome cold-start issues. [Pulumi tutorials]({{< relref "/registry" >}}) can get you started with serverless deployments.
```

Comment:
i did not know what to replace this with. can someone provide a suggestion?

Reply:
Ugh, I think our original intent was to keep `pulumi.com/docs/tutorials` around specifically because there was no logical mapping from that page into something in the new registry environment. I don't have a good answer here, but I'm down to figure something out moving forward.